src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/spreadsheet/CCDConfigValidator.java

```diff
@@ -261,9 +259,30 @@ && fieldDoesNotHaveAValidMapping(ccdFieldAttributes, field))) {
         return errors;
     }
 
+    private String resolveSimpleNameFromCCDFieldType(String ccdFieldType) {
+        if (fieldTypesMap.containsKey(ccdFieldType)) {
+            return fieldTypesMap.get(ccdFieldType);
+        } else {
+            return resolveSimpleNameFromPattern(ccdFieldType);
+        }
+    }
+
+    private boolean doesNotMatchFieldSimpleName(String expectedClassSimpleName, Class clazz) {
+        return !expectedClassSimpleName.toLowerCase().equals(clazz.getSimpleName().toLowerCase());
+    }
+
+    private String resolveSimpleNameFromPattern(String ccdFieldType) {
+        if (ccdFieldType != null && ccdFieldType.startsWith(FR_CCD_FIELD_TYPE_PREFIX)) {
+            return ccdFieldType.substring(FR_CCD_FIELD_TYPE_PREFIX.length());
+        } else {
+            return null;
+        }
+    }
```

Comment:
I like this.  

If somebody new (or somebody forgets), they may forget to create a field with the FR_ prefix.   Rather than ask them to debug this validator, could we put a helpful log in the else condition?

Something like "couldn't find your field, don't forget your FR_ prefix'

Reply:
Brilliant thanks.